spec/controllers/article_finder_controller_spec.rb

```diff
@@ -3,25 +3,27 @@
 require 'rails_helper'
 
 describe ArticleFinderController do
-  describe 'index' do
-    it 'renders' do
-      get :index
-      expect(response.status).to eq(200)
+  if Features.enable_article_finder?
```

Comment:
Instead of skipping these tests, you should do `allow(Features).to receive(:enable_article_finder?).and_return(true)` in a `before` block.